cypress/cypress.json

```diff
@@ -6,18 +6,13 @@
     "supportFile": "test/cypress/support/index.ts",
     "pluginsFile": "test/cypress/plugins/index.ts",
 
-    "defaultCommandTimeout": 90000,
+    "defaultCommandTimeout": 30000,
```

Comment:
Looks good, but I added a small suggestion in https://github.com/cloudify-cosmo/cloudify-stage/pull/1400#pullrequestreview-678449248

Reply:
Seems like with small change to user flow spec (https://github.com/cloudify-cosmo/cloudify-stage/pull/1418) we are able to go down to 15s.

Reply:
Awesome :smile: :+1: